Updating Faucet, NFT Tutorial, and Sending Transaction Document (#5225)
ethereum/ethereum-org-website: src/content/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md

### 2\. Request ETH from the Rinkeby faucet {#request-eth-from-rinkeby-faucet}

> Follow the instructions on the [Rinkeby faucet](https://www.rinkebyfaucet.com/) to receive Eth. You will have to share on social media for this specific faucet. Make sure to include your **Rinkeby** Ethereum address (from Metamask) and not another network. After following the instructions, double check that you’ve received the ETH in your wallet.

Reviewer: We should specify this is Alchemy's faucet. It also doesn't require social media validation so removed that.

```suggestion
Follow the instructions on the [Alchemy Rinkeby faucet](https://www.rinkebyfaucet.com/) to receive ETH. Make sure to include your **Rinkeby** Ethereum address (from Metamask) and not another network. After following the instructions, double-check that you’ve received the ETH in your wallet.
```
